docs/custom-stack.asciidoc

```diff
@@ -48,6 +48,9 @@ var apm = require('elastic-apm-node').start({
   // Use if APM Server requires a token
   secretToken: '',
 
+  // Use if APM Server uses API keys for authentication
+  apiKey: '',
+
```

Comment:
Not required, just throwing this out there. Could combine secretToken and apiKey into something like.

```
  // If the APM Server requires authentication via a token or API key use one of:
  secretToken: '',
  apiKey: '',
```